test/manifold_test.cpp

```diff
@@ -711,4 +711,28 @@ TEST(Manifold, PinchedVert) {
   EXPECT_FALSE(touch.IsEmpty());
   EXPECT_EQ(touch.Status(), Manifold::Error::NoError);
   EXPECT_EQ(touch.Genus(), 0);
-}
\ No newline at end of file
+}
+
+TEST(Manifold, Hull) {
```

Comment:
I'd like to see another test with two high-res spheres - one before the precision limit, testing that the number of verts is the same after hulling as before, and another past the limit where it's less (but the volume should still be close).